C_sharp_exercises_2/C_sharp_exercises_2/Exercises4.cs

```diff
@@ -0,0 +1,71 @@
+﻿using C_sharp_exercises_2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+//Exercise 4:
+//Create a class Book which should have author, title and createdDate properties.
+//Create 10 instances of Book class.
+//1.Filter latest 5 created Books.
+//2. Group the books which authors are same person
+//3. Find Books which title starts with A letter
+namespace C_sharp_exercises_2
+{
+    class Book
+    {
+        public string author;
+        public string title;
+        public DateTime BookDta;
+        public Book(string name, string title, DateTime BookDta)
```

Comment:
why you get the datetime? you can assign it in constructor when its created

Reply:
but also i have to pass data to properties, can i do it at the same time ? or am I misunderstanding the question